### DISCUSSION TITLE
Duda sobre informe

### DISCUSSION THREAD

**Discussion by gabo1243** [IIC2133-PUC/2024-1#159] [Tarea 2] [answered]
Hola, para el informe ¿qué tanto detalle hay que mencionar con respecto al hashing? ¿Hay que poner la fórmula completa o solo describirla superficialmente y tratarla como "función hash"?

Gracias de antemano.

**Comment by sachondo7** [comment, marked as answer]
Hola, se espera que puedas explicar el hashing entero como función, aunque de forma breve

Para la P1 y el rolling hash, lo ideal es que mencionen cual es la base y el tipo de input. Después en los subarboles, como une los hash de diferentes alturas y calcula hash a partir de nodos. 
Asique ahí tienes que encontrar un equilibrio en la cantidad de páginas y la info que agregas. Recomiendo ser lo más específico posible sin pasarse del límite y sonar redundante
